esp/services/ecldirect/EclDirectService.cpp

```diff
@@ -49,6 +49,9 @@ EclDirectWUExceptions::EclDirectWUExceptions(IConstWorkUnit& cw)
         e->setLineNo(item.getExceptionLineNo());
         e->setColumn(item.getExceptionColumn());
         e->setActivity(item.getActivityId());
+        if (item.getPriority())
```

Comment:
No ESP will handle the versioning for you.  You can make things conditional if you think they are expensive enough to warrant the time saved.